siliconcompiler/core.py

```diff
@@ -4061,7 +4064,10 @@ def _reset_flow_nodes(self, flow, steplist, indexlist):
                 for record in self.getkeys('record'):
                     self._clear_record(step, index, record)
             elif os.path.isfile(cfg):
-                self.set('flowgraph', flow, step, index, 'status', NodeStatus.SUCCESS)
+                chip_node_status = Chip('node_succeess')
+                chip_node_status._read_manifest(cfg)
+                node_status = chip_node_status.get('flowgraph', flow, step, index, 'status')
```

Comment:
Could this be done more cleanly with a `Schema` object? (bit of a nit-pick, but don't need the fake design name then for example)

Reply:
Oh of course I was thinking initially that directly accessing the Schema might be discouraged or sth.